### DISCUSSION TITLE
Requesting for add me in the team

### DISCUSSION THREAD

**Discussion by ghost** [ROM-builders/temporary#16828] [Q&A]
Hello Sir,  @Apon77 
I have successfully build with PR
[Derpfest](https://github.com/ROM-builders/temporary/pull/16779)
[Cherish](https://github.com/ROM-builders/temporary/pull/16883)
And [derpfest](https://cirrus-ci.com/build/4514142195810304) [Cherish](https://cirrus-ci.com/build/6623634341494784)
Please add me in team
Here's my repository: 
[CherishOS](https://github.com/rinto02/RMX2020-CherishOS-tiramisu-rinto02)
[Derpfest](https://github.com/rinto02/RMX2020-DerpFest-AOSP-13-rinto02)

Distribute the rom with [telegram](https://t.me/@RealmeC310a)

Thanks

**Comment by Apon77** [comment]
please follow #learn in telegram group
